features/facebookImport/src/model/entities/facebook-account.js

```diff
@@ -23,6 +23,7 @@ class FacebookAccount {
         this._adminRecords = [];
         this._accountSessionActivities = [];
         this._postReactions = [];
+        this._posts = [];
```

Comment:
Do these posts include reactions? In that case, can we maybe skip the previous attribute?

Reply:
They are separate. For post reactions (and also comments) we do not have the post. We just know that a liked happened and the account name that the liked post belongs to. We could use that later on to better model this (associate likes to related accounts) but for now all we need is the timestamp.